templates/community-viz/src/index.js

```diff
@@ -1,38 +1,122 @@
 const dscc = require('@google/dscc');
 
+
+
 // import the local data
 import * as local from './localData.js';
 
 // change this to 'true' for local development
 // change this to 'false' before deploying
-const LOCAL = false;
+const LOCAL = true;
+
+const README = `
+<h2>Community Viz Local Development</h2>
+<p>
+To edit the visualization and see the changes here, edit <code>src/index.js</code>.
+</p>
+<p>
+To change the data:
+<ol>
+<li>Make sure the config in <code>src/index.json</code> is correct</li>
+<li>Run <code>npm run message</code> from the command-line </li>
+<li>Use your "dev bucket" as the component ID and deploy the visualization in Data Studio</li>
+<li>Copy the printed message + code into <code>src/localData.js</code></li>
+</ol>
+</p>
+
+<p style="margin-bottom:1cm;">
+
+</p>
+
+`
 
 if (LOCAL) {
   setTimeout(() => {
     window.postMessage(local.message, '*');
   }, 1000);
 }
 
-// Write Viz Code here
+// write viz code here
```

Comment:
I think this would be better as a giant multi-line REPLACE ME so it's crystal clear what the developer should delete vs keep.